app/templates/thank-you.html

```diff
@@ -9,19 +9,37 @@ <h1 class="saturn">Submission Successful</h1>
     <p>Thank you for submitting the {{ survey_title }}, your responses have been successfully received.</p>
 </div>
 
-<div class="panel panel--simple panel--success panel--spacious">
-    <p class="mars">Transaction ID (Please quote this reference for any support queries)</p>
-    {% if meta.respondent %}
-    <ul class="list list--boxes">
-        {%for id_part in meta.respondent.tx_id%}<li class="list__item">{{id_part}}</li>{% endfor %}
-    </ul>
-    {% endif %}
-    <p class="u-mb-no mars">You may wish to save or print this page for your records.</p>
+<div class="u-mb-s">
+    <div class="panel panel--simple panel--success panel--spacious">
+        <p class=""mars>Your answers were submitted for <b>{{meta.respondent.ru_name}}</b> on {{meta.survey.submitted_time}}</p>
+        <p class="mars">Transaction ID: <b>{{meta.respondent.tx_id}}</b></p>
+    </div>
+</div>
+
+<div class="u-mb-s">
+    <p>Your answers will be processed in the next few weeks.
+        We may contact you to query your answers via phone or secure message</p>
 </div>
 
-{{answer_guidance}}
+{% block view_submission %}
+{% if meta.survey.submission_validity > 0 %}
+    <div class="u-mb-s">
+    {% if view_submission_url %}
+        <a class="btn btn--primary btn--lg" data-qa="btn-view-submission" href={{view_submission_url}}>View and print a copy of your answers</a>
```

Comment:
Links shouldn't look like buttons.

Reply:
@hamishtaplin, @armstrongb If we don't use a button I need some advice on how to make this link a primary feature of the page please.